.prospector.yml

```diff
@@ -10,16 +10,18 @@ test-warnings: true
 member-warnings: false
 
 pyroma:
-    run: true
+  run: true
 
 pep8:
-    full: true
+  full: true
+  # ignore rules that conflict with ruff formatter
+  disable: ['E203', 'E501', 'W503']
```

Comment:
Is there some kind of reference that you could add as a comment here?

Reply:
Done in 7b0f8cd